stripe/res/layout/stripe_googlepay_button_black.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:clickable="true"
+    android:focusable="true"
+    android:layout_width="match_parent"
+    android:layout_height="48sp"
+    android:background="@drawable/stripe_googlepay_button_no_shadow_background_black"
+    android:paddingTop="2sp"
```

Comment:
sp for layout size and padding -- is this intentional?

Reply:
This is intentional, copied directly from Google Pay's source.